examples/config/config_parser.py

```diff
@@ -0,0 +1,31 @@
+import yaml
+import platform
+
+print(platform.system())
+
+class parser:
+    def __init__(self,filepath):
```

Comment:
Good practice to put spaces between comma and next parameter:
```
 def __init__(self, filepath):
```

Reply:
Thanks!!